### ISSUE TITLE
[QUESTION] <Brief Description> convnext-large-ltdetr-coco train log

### ISSUE THREAD

**Issue by chcchc97** [lightly-ai/lightly-train#766]
Hi, I would like to ask about the specific hyperparameters used to train dinov3/convnext-large-ltdetr-coco on COCO. Is it possible for you to share the training logs?

**Comment by liopeer** [comment]
Hey, unfortunately I can't recover the exact logs (only the experiment logs that we have in our experiment tracking). Are you unable to reproduce the results?

**Comment by chcchc97** [comment]
@liopeer Thank you very much for your previous response. 

To help us successfully replicate your excellent results, would you be so kind as to share the specific hyperparameter settings used in this experiment? We would deeply appreciate it if you could provide details on the following:

Learning Rate Strategy: Initial learning rate, decay policy (e.g., Cosine/Step), and warmup epochs.

Data Augmentation: Specific augmentation techniques applied (e.g., mixup, mosaic, random cropping) and their intensities/probabilities.

Optimizer Configurations: Choice of optimizer (e.g., AdamW, SGD), weight decay, and momentum.

Training Schedule: Batch size and total number of epochs.

**Comment by chcchc97** [comment]
> [@liopeer](https://github.com/liopeer) Thank you very much for your previous response.
> 
> To help us successfully replicate your excellent results, would you be so kind as to share the specific hyperparameter settings used in this experiment? We would deeply appreciate it if you could provide details on the following:
> 
> Learning Rate Strategy: Initial learning rate, decay policy (e.g., Cosine/Step), and warmup epochs.
> 
> Data Augmentation: Specific augmentation techniques applied (e.g., mixup, mosaic, random cropping) and their intensities/probabilities.
> 
> Optimizer Configurations: Choice of optimizer (e.g., AdamW, SGD), weight decay, and momentum.
> 
> Training Schedule: Batch size and total number of epochs.

@yutong-xiang-97 Hi, yutong, could you please answear this quesitons?

**Comment by liopeer** [comment]
Training parameters should have matched exactly what's implemented here:
https://github.com/lightly-ai/lightly-train/blob/5e969241f4e5b36c481218f8d9a36ce8b1b6074c/src/lightly_train/_task_models/dinov3_ltdetr_object_detection/train_model.py#L97-L120

The augmentations are configure here: 
https://github.com/lightly-ai/lightly-train/blob/5e969241f4e5b36c481218f8d9a36ce8b1b6074c/src/lightly_train/_task_models/dinov3_ltdetr_object_detection/transforms.py#L234-L249
